### PR TITLE
1.19.2 Support and Clean up

### PR DESCRIPTION
**Pull request by JT122406**

### CODE REVIEW

File: src/main/java/ca/spottedleaf/starlight/Starlight.java [hunk 1]
```diff
@@ -10,9 +10,7 @@ public final class Starlight {
 
     public Starlight() {
         // starlight is network compatible both ways
-        ModLoadingContext.get().registerExtensionPoint(IExtensionPoint.DisplayTest.class, () -> {
-            return new IExtensionPoint.DisplayTest(() -> NetworkConstants.IGNORESERVERONLY, (a, b) -> true);
-        });
+        ModLoadingContext.get().registerExtensionPoint(IExtensionPoint.DisplayTest.class, () -> new IExtensionPoint.DisplayTest(() -> NetworkConstants.IGNORESERVERONLY, (a, b) -> true));
```
Comment (olijeffers0n) [inline src/main/java/ca/spottedleaf/starlight/Starlight.java]: This is intentional as your “fix” is hard to read :)

File: src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java [hunk 2]
```diff
@@ -178,28 +178,24 @@ public void clientUpdateLight(final LightLayer lightType, final SectionPos pos,
         // data storage changed with new light impl
         final ChunkAccess chunk = this.getLightEngine().getAnyChunkNow(pos.getX(), pos.getZ());
         switch (lightType) {
-            case BLOCK: {
-                final SWMRNibbleArray[] blockNibbles = this.blockLightMap.computeIfAbsent(CoordinateUtils.getChunkKey(pos), (final long keyInMap) -> {
-                    return StarLightEngine.getFilledEmptyLight(this.lightEngine.getWorld());
-                });
+            case BLOCK -> {
```
Comment (olijeffers0n) [inline src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java]: Again, too long

File: src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java [hunk 3]
```diff
@@ -178,28 +178,24 @@ public void clientUpdateLight(final LightLayer lightType, final SectionPos pos,
...
                     this.lightEngine.getLightAccess().onLightUpdate(LightLayer.BLOCK, pos);
                 }
                 break;
             }
-            case SKY: {
-                final SWMRNibbleArray[] skyNibbles = this.skyLightMap.computeIfAbsent(CoordinateUtils.getChunkKey(pos), (final long keyInMap) -> {
-                    return StarLightEngine.getFilledEmptyLight(this.lightEngine.getWorld());
-                });
+            case SKY -> {
+                final SWMRNibbleArray[] skyNibbles = this.skyLightMap.computeIfAbsent(CoordinateUtils.getChunkKey(pos), (final long keyInMap) -> StarLightEngine.getFilledEmptyLight(this.lightEngine.getWorld()));
```
Comment (olijeffers0n) [inline src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java]: Too long

**Review by olijeffers0n — COMMENTED** [review]
Few Changes I disagree with :)

File: src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java (hunk 2 above)
Comment (JT122406) [reply src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java]: This is a Java 14 enhanced Switch its supposed to make it look nicer

Comment (JT122406) [reply src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java]: but I can revert that if you guys want to

Comment (olijeffers0n) [reply src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java]: I meant to click the line below - my bad 😣

Comment (JT122406) [reply src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java]: all good lol

**Comment by JT122406** [comment]
I have addressed the things needing changes however I have a concern for the LevelLightEngineMixin 

if (((Object)this).getClass() != LevelLightEngine.class) {
            throw new IllegalStateException("This hook is for the CLIENT ONLY");
        }  this is always true so everything below it never runs

**Comment by Spottedleaf** [comment]
From what I can tell, 1.19.2 is supported already

**Pull request closed without merging**